src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx

```diff
@@ -255,4 +255,13 @@
   <data name="FixNumberedComments" xml:space="preserve">
     <value>Fix numbered comments</value>
   </data>
+  <data name="DefaultableTypeShouldHaveDefaultableFieldsDescription" xml:space="preserve">
+    <value>Defaultable types should have defaultable fields</value>
```

Comment:
Don't think this description is much value add over the message. Remove?

Reply:
I like to keep both in case one gets expanded in the future.